pkg/controller/oneagent/istio.go

```diff
@@ -181,65 +191,93 @@ func (r *ReconcileOneAgent) removeIstioConfigurationForVirtualService(
 	return del
 }
 
-func (r *ReconcileOneAgent) reconcileIstioCreateConfigurations(
-	instance *dynatracev1alpha1.OneAgent,
-	ic *versionedistioclient.Clientset,
-	comHosts []dtclient.CommunicationHost,
-	role string, logger logr.Logger) bool {
+func (r *ReconcileOneAgent) reconcileIstioCreateConfigurations(instance *dynatracev1alpha1.OneAgent,
+	istioClient *versionedistioclient.Clientset, communicationHosts []dtclient.CommunicationHost, role string,
+	logger logr.Logger) bool {
 
-	created := false
+	crdProbe := r.verifyIstioCrdAvailability(instance, logger)
+	if crdProbe != probeTypeFound {
+		logger.Info("istio: failed to lookup CRD for ServiceEntry/VirtualService: Did you install Istio recently? Please restart the Operator")
+		return false
```

Comment:
Can't we _just_ exit the process and forcing a _restart_ of the operator process?

This of cause has a change of `CrashLoopBackOff` if this path in the code happens to be called more than just in that condition ;)